lib/bolt/outputter/human.rb

```diff
@@ -165,6 +165,8 @@ def print_plan_info(plan)
       def print_plan_result(plan_result)
         if plan_result.value.nil?
           @stream.puts("Plan completed successfully with no result")
+        elsif plan_result.value.is_a? Bolt::ApplyFailure
```

Comment:
Lets create a ticket to revisit human output for plans that error. I suspect there is something standard we can do here.

Reply:
Added BOLT-716.